src/postprocessor.rs

```diff
@@ -54,6 +55,7 @@ pub fn lookup(name: &str) -> Result<Box<dyn Postprocessor>> {
         "ingest-ns" => Ok(Box::new(AttachIngresTs {})),
         "length-prefixed" => Ok(Box::new(LengthPrefix::default())),
         "gelf-chunking" => Ok(Box::new(Gelf::default())),
+        "textual-length-prefix" => Ok(Box::new(TextualLength::default())),
```

Comment:
maybe also add `textual-length-prefixed` as lookup key here, so we are consistent with the `length-prefixed` postprocessor.